src/Microsoft.Data/StringColumn.cs

```diff
@@ -287,5 +289,18 @@ public override GroupBy GroupBy(int columnIndex, DataFrame parent)
                 throw new NotImplementedException(nameof(TKey));
             }
         }
+
+        protected internal override void AddDataViewColumn(DataViewSchema.Builder builder)
+        {
+            builder.AddColumn(Name, TextDataViewType.Instance);
+        }
+
+        protected internal override Delegate GetDataViewGetter(DataViewRowCursor cursor)
+        {
+            return CreateValueGetterDelegate(cursor);
+        }
+
+        private ValueGetter<ReadOnlyMemory<char>> CreateValueGetterDelegate(DataViewRowCursor cursor) =>
+            (ref ReadOnlyMemory<char> value) => value = this[cursor.Position].AsMemory();
```

Comment:
This might be interesting once `ArrowStringColumn` goes in.

Reply:
Yes, when our 2 changes come in, one will have to merge with the other.